extensions/dicom-seg/src/components/SegmentationSettings/SegmentationSettings.js

```diff
@@ -0,0 +1,169 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Range } from '@ohif/ui';
+
+import './SegmentationSettings.css';
+
+const SegmentationSettings = ({ configuration, onBack, onChange }) => {
+  const [state, setState] = useState({ ...configuration });
```

Comment:
Should we destructure here so we don't put anything in state by accident?

Reply:
configuration is tied to cornerstone-tools seg module, right?
So this is our first bit of reactive state, `onChange` updates cornerstone-tools seg module.

1. Do we need to worry about anything else updating it, but we aren't notified here?
2. If parent is pulling it to pass to this component, why not hoist state to it (or move query to here?)

Reply:
Done!